analysis_service/base/models.py

```diff
@@ -0,0 +1,206 @@
+from datetime import datetime, timedelta
+from pytz import UTC
+from django.db import models
+from django.contrib.auth.models import User
+
+from analysis_service.base.util import provisioning, scheduling
+
+
+class Cluster(models.Model):
```

Comment:
AFAIK it's not used for anything, it just tags the resources with the name so we can match up the instance or cluster with a job or a person in the AWS Console. It's convenient for housekeeping / debugging.

It should be about the same in ATMO - the name would appear in your list so you have some indication of what a given cluster / instance was doing.